Tests/UnitTests/Errors/GenericErrorViewModelTests.swift

```diff
@@ -38,19 +38,31 @@ extension GenericErrorViewModelTests {
         sut.primaryButtonViewModel.action()
         XCTAssertTrue(didCallButtonAction)
         XCTAssertEqual(mockAnalyticsService.eventsLogged.count, 1)
-        let event = ButtonEvent(textKey: "app_closeButton")
+        let event = LinkEvent(textKey: "app_closeButton",
+                              linkDomain: AppEnvironment.oneLoginBaseURL,
+                              external: .false)
         XCTAssertEqual(mockAnalyticsService.eventsLogged, [event.name.name])
         XCTAssertEqual(mockAnalyticsService.eventsParamsLogged["text"], event.parameters["text"])
         XCTAssertEqual(mockAnalyticsService.eventsParamsLogged["type"], event.parameters["type"])
+        XCTAssertEqual(mockAnalyticsService.eventsParamsLogged["link_domain"],
+                       event.parameters["link_domain"])
+        XCTAssertEqual(mockAnalyticsService.eventsParamsLogged["external"],
+                       event.parameters["external"])
     }
     
     func test_didAppear() throws {
         XCTAssertEqual(mockAnalyticsService.screensVisited.count, 0)
         sut.didAppear()
         XCTAssertEqual(mockAnalyticsService.screensVisited.count, 1)
-        let screen = ScreenView(screen: ErrorAnalyticsScreen.generic,
-                                titleKey: "app_somethingWentWrongErrorTitle")
+        let screen = ErrorScreenView(id: ErrorAnalyticsScreenID.generic.rawValue,
+                                     screen: ErrorAnalyticsScreen.generic,
+                                     titleKey: "app_somethingWentWrongErrorTitle",
+                                     reason: "reason")
```

Comment:
Happy to approve on the basis that we're still waiting to hear back from the analytics team on this parameter?

Reply:
I've not heard back from them at all, but I've now passed in the `error.localizedDescription` to `createGenericError` which is probably a more robust solution moving forward anyway - but happy to revert and wait if we think that's what is best!